src/app/js/formats/streamgraph/Streamgraph.js

```diff
@@ -308,22 +319,38 @@ class Streamgraph extends PureComponent {
             legendItemContainer
                 .append('text')
                 .attr('class', `${css(styles.legendItemText)}`)
-                .text(cutStr(element.name));
+                .text(cutStr(element.name, column_number));
 
             const legendItemTooltip = legendItemContainer
                 .append('span')
                 .attr('class', `${css(styles.legendItemTooltip)}`)
                 .text(element.name);
 
             legendItemContainer
-                .on('mouseover', (d, i) => {
+                .on('mouseover', d => {
```

Comment:
eslint says that `d` is not used.

```suggestion
                .on('mouseover', () => {
```

Reply:
Oh yeah this `() => { ... }` syntax was the thing I missed and needed, thanks !